src/Pos/Wallet/Web/Api.hs

```diff
@@ -241,6 +241,15 @@ type NewPayment =
     :> Capture "amount" Coin
     :> WRes Post CTx
 
+type TxFee =
+       "txs"
+    :> "fee"
+    :> DCQueryParam "passphrase" CPassPhrase
```

Comment:
@akegalj password is needed for creating of tx, I just used existed functions.
Fee depends on size of created tx.
@Martoon-00 we can't create a tx without a password, can we?